snapcraft.yaml

```diff
@@ -190,11 +193,32 @@ parts:
         echo "Preparing istio"
         cp $KUBE_SNAP_BINS/istioctl .
         cp -r $KUBE_SNAP_BINS/istio-yaml ./actions/istio
+        # Add bash completion for istioctl. This is located here to make sure it is only added when istio is added.
+        if [ -f istioctl ]; then
```

Comment:
Could we also skip the file existence checks? If any executable (`istioctl` or `kubectl`) is missing calling it will fail stopping snapcraft build. This is the reason we have `set -eu` at the top.

Reply:
Fixed